packages/replay-internal/test/unit/util/createPerformanceEntry.test.ts

```diff
@@ -66,16 +72,134 @@ describe('Unit | util | createPerformanceEntries', () => {
     it('works with an LCP metric', async () => {
       const metric = {
         value: 5108.299,
+        rating: 'good' as const,
         entries: [],
       };
 
       const event = getLargestContentfulPaint(metric);
 
       expect(event).toEqual({
-        type: 'largest-contentful-paint',
+        type: 'web-vital',
```

Comment:
just to clarify, this will not break anything, right?

Reply:
Currently LCP breadcrumbs are hidden in Replays so it shouldn't